packages/engine-client/src/utils/queryDataMappers.ts

```diff
@@ -255,3 +259,25 @@ export function mapEngineMarketPrice(
     productId: baseResponse.product_id,
   };
 }
+
+export function mapEngineServerDepositRates(
+  baseResponse: EngineServerMinDepositRatesResponse,
+): GetEngineMinDepositRatesResponse {
+  const minDepositRates: Record<number, EngineMinDepositRate> = mapValues(
+    baseResponse.min_deposit_rates,
+    mapEngineMinDepositRate,
+  );
+
+  return {
+    minDepositRates,
```

Comment:
Backend nests in an object, but do we need to do this? or could we just return the `Record<number, EngineMinDepositRate>`?

Reply:
We are a bit inconsistent with this unfortunately :( ex. `getAllMarkets` just returns `markets` directly

Reply:
Looking at this more let's just return `minDepositRates` directly, we can switch the outliers over some other time